src/views/Project/Details/General/index.js

```diff
@@ -38,11 +38,13 @@ const propTypes = {
     projectServerData: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
     setErrorProjectDetails: PropTypes.func.isRequired,
     projectId: PropTypes.number.isRequired,
+    disabled: PropTypes.bool.isRequired,
```

Comment:
Required and default props both are in place.
How do you mean?

Reply:
haha bhetayecha, tara farak pardaina
runtime ma isRequired cha bhane console ma warning aucha, tetti nai ho 

Malai afai lai khojna alchi lagirathyo :D